### DISCUSSION TITLE
Is v0.2 compatible with torch==2.3 and CUDA version 12.1?

### DISCUSSION THREAD

**Discussion by Dr-Left** [flashinfer-ai/flashinfer#687] [Q&A] [answered]
Is FlashInfer v0.2 compatible with torch==2.3 and CUDA version 12.1?

**Comment by yzh119** [comment, marked as answer]
Available at https://flashinfer.ai/whl/cu121/torch2.3/flashinfer/
FA3 template requires cuda 12.4 btw.
